common/changes/@cityofzion/bs-neo-legacy/CU-86a89ahfx_2025-05-01-00-28.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@cityofzion/bs-neo-legacy",
+      "comment": "Adjust transfers to handle scenarios that have a transaction size beyond 1024",
```

Comment:
Change `1024` to `1024kb`.

Reply:
that is incorrect. The limit is 1024 bytes [ref1](https://github.com/nspcc-dev/neo-go/blob/fae1d562e7852e57c9f2077c6fc913c3e955204b/pkg/config/protocol_config.go#L40-L41), [ref2](https://github.com/nspcc-dev/neo-go/blob/fae1d562e7852e57c9f2077c6fc913c3e955204b/config/protocol.mainnet.yml#L40)